nva-datamodel-java/src/main/java/no/unit/nva/model/instancetypes/artistic/film/MovingPictureSubtypeEnum.java

```diff
@@ -1,8 +1,9 @@
 package no.unit.nva.model.instancetypes.artistic.film;
 
 import com.fasterxml.jackson.annotation.JsonValue;
+import no.unit.nva.model.instancetypes.artistic.WithType;
 
-public enum MovingPictureSubtypeEnum {
+public enum MovingPictureSubtypeEnum implements WithType {
```

Comment:
I do not think that the enum implements an interface WithType but an interface IsType.